dev/tools/update_icons.dart

```diff
@@ -28,6 +28,82 @@ const Map<String, String> kIdentifierRewrites = const <String, String>{
   'class': 'class_',
 };
 
+final Set<String> kMirroredIcons = new Set<String>.from(<String>[
+  // This list is obtained from:
+  // http://google.github.io/material-design-icons/#icons-in-rtl
+  'arrow_back',
+  'arrow_back_ios',
+  'arrow_forward',
+  'arrow_forward_ios',
+  'arrow_left',
+  'arrow_right',
+  'assignment',
+  'assignment_return',
+  'backspace',
+  'battery_unknown',
+  'call_made',
+  'call_merge',
+  'call_missed',
+  'call_missed_outgoing',
+  'call_received',
+  'call_split',
+  'chevron_left',
+  'chevron_right',
```

Comment:
Crazytown

Reply:
Yeah. I considered making new versions of the icons with more direction-agnostic names, and leaving these as non-flipped, but I was worried that that wouldn't be forward-compatible if they ever switch to doing this in the font.

Reply:
(If they decide not to change this in the font, then we can revisit.)